docs/src/includes/_globals.md

```diff
@@ -147,6 +147,15 @@ Clears a callback that is specific to a screen.
 
 Clears a previously set callback
 
+### register_option_callback
+
+
+> Search script examples for [register_option_callback](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=register_option_callback)
+
+#### nil register_option_callback(string name, function callback)
+
+Add custom options using the window drawing functions. Your callback will be called with a [GuiDrawContext](#GuiDrawContext) as a parameter and everything drawn in it will be rendered in the options window and the return value saved to `options[name]`.
```

Comment:
Can you format it like i did in my PR "The callback signature is" in separate line?

Reply:
Yep